sources/asia/ph/Arenda-OAM.geojson

```diff
@@ -0,0 +1,110 @@
+    {
+      "type": "Feature",
+      "properties": {
+        "id": "Arenda_OAM",
+        "name": "Lupang Arenda, Taytay Drone Imagery",
+        "type": "tms",
+        "url": "https://tiles.openaerialmap.org/5d25d7bf161a790005c03e6e/0/5d25d7bf161a790005c03e6f/{zoom}/{x}/{y}.png",
+        "min_zoom": 16,
+        "max_zoom": 22,
+        "country_code": "PH",
+        "license": "CC-BY 4.0",
```

Comment:
Per https://github.com/osmlab/editor-layer-index/blob/ec3561b31f161cbf5452fdd9edf12f89b1f05901/schema.json#L68-L70 this must be a [SPDX identifier](https://spdx.org/licenses/). so should be `CC-BY-4.0`. You're missing a `-`. That's why the check is failing.

Reply:
Oh thank you!